lsp-ltex.el

```diff
@@ -367,10 +452,10 @@ This file is use to activate the language server."
 (lsp-register-custom-settings
  '(("ltex.enabled" lsp-ltex-enabled)
    ("ltex.language" lsp-ltex-language)
-   ("ltex.dictionary" lsp-ltex-dictionary)
-   ("ltex.disabledRules" lsp-ltex-disabled-rules)
+   ("ltex.dictionary" lsp-ltex--combined-dictionary)
+   ("ltex.disabledRules" lsp-ltex--combined-disabled-rules)
+   ("ltex.hiddenFalsePositives" lsp-ltex--combined-hidden-false-positives)
    ("ltex.enabledRules" lsp-ltex-enabled-rules)
-   ("ltex.hiddenFalsePositives" lsp-ltex-hidden-false-positives)
```

Comment:
Maybe revert the order? So it's matches to their [settings](https://valentjn.github.io/ltex/settings.html) page.

Reply:
Ok!